recipe-server/client/actions/preference-rollout/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "preference-rollout",
+    "version": "0.0.1",
+    "private": true,
+    "main": "./index.js",
+    "normandy": {
+        "driverVersion": "1.x",
+        "argumentsSchema": {
+            "$schema": "http://json-schema.org/draft-04/schema#",
+            "title": "Change a preference for a portion of the population",
+            "type": "object",
+            "required": [
+                "message"
+            ],
+            "properties": {
+                "name": {
+                    "description": "Rollout name",
+                    "type": "string",
+                    "default": ""
+                },
+                "preferenceName": {
+                    "description": "Preference to change",
+                    "type": "string",
+                    "default": ""
+                },
+                "preferenceValue": {
+                    "description": "Value to set preference to.",
+                    "type": ["string", "number", "boolean"],
+                    "default": ""
+                },
+                "preferenceBranch": {
+                    "descript": "Controls whether the default or user value of the preference is modified",
+                    "type": "string",
+                    "enum": ["user", "default"],
+                    "default": "default"
+                },
+                "branchConfirmation": {
```

Comment:
Does this need to be part of the arguments? I don't think any consumer of this will need this value, and it is mostly a UI treatment. I think it would be better to not include it here if that is possible.